tests/framework/resource/test_manager.py

```diff
@@ -42,28 +43,31 @@ def initializer(self, _simulant_data: SimulantData) -> None:
 
 
 @pytest.mark.parametrize(
-    "resource_class, type_string, is_initializer",
+    "resource_class, init_args, type_string, is_initializer",
     [
-        (Pipeline, "value", False),
-        (ValueSource, "value_source", False),
-        (MissingValueSource, "missing_value_source", False),
-        (ValueModifier, "value_modifier", False),
-        (Column, "column", True),
-        (NullResource, "null", True),
+        (Pipeline, ["foo"], "value", False),
+        (ValueSource, [Pipeline("foo"), lambda: 1], "value_source", False),
+        (ValueModifier, [Pipeline("foo"), lambda: 1], "value_modifier", False),
+        (Column, ["foo"], "column", True),
+        (NullResource, ["foo"], "null", True),
     ],
-    ids=lambda x: {x.__name__ if isinstance(x, type) else x},
+    ids=lambda x: [x.__name__ if isinstance(x, type) else x],
```

Comment:
How is type working here?

Reply:
My understanding is that `type` is a build-in meta-class that classes (but _not_ instances) inherit from.

```
class SomeClass:
    pass

isinstance(SomeClass, type)
True

isinstance(SomeClass(), type)
False
```